### DISCUSSION TITLE
With the operation of the Lotus daemon and the increase in the amount of data, will the memory used also increase?

### DISCUSSION THREAD

**Discussion by YuXiaoCoder** [filecoin-project/lotus#9945] [General Q&A]
Lotus Version:
```bash
./lotus version
Daemon:  1.18.0+mainnet+git.1f6b30c14+api1.5.0
Local: lotus version 1.18.0+mainnet+git.1f6b30c14
```

I run two nodes with the same configuration. One node has 393G of data, and the corresponding memory used is 54G, while the other node has 168G of data, and the corresponding memory used is only 11G. Is there any relationship between them?

**Comment by Reiers** [comment]
Hi @YuXiaoCoder 👋 

First I would recommend updating Lotus to `v1.19.0` - always good practise to be on the latest stable release. 

So let's see:

- Does the nodes run splitstore? 
- Does the nodes have the exact same hardware?
- Have the nodes been restarted after the network upgrade?

My nodes is using around 11G, all running _splitstore_. The only time I saw it increase in RAM usage was under the network upgrade, and all it took was a restart for it to go down back to normal.

**Reply by YuXiaoCoder** [reply]
@Reiers 
I will update Lotus to v1.19.0 later,
1. The current node is not running splitstore, I see that there is a v2 version of splitstore, but it is in beta state.
2. The two nodes use the same hardware, there is no difference
3. When the network was upgraded to Filecoin network v17, we restarted the nodes

**Reply by YuXiaoCoder** [reply]
After I upgraded and enabled splitstore, the memory usage is still high
![image](https://user-images.githubusercontent.com/26297513/209769216-d808e415-ff45-41da-aa8b-3cdf1cebf797.png)

**Reply by lcgogo** [reply]
meet the same issue, 1.19.0 lotus nodes memory usage may increase very high and cause the server halt.

normally the node use 12 GB memory but it may up to more than 40-50 GB and crush by itself.

<img width="793" alt="图片" src="https://user-images.githubusercontent.com/8175573/211153947-f8c564de-5df4-4f2d-aa5c-bde68d5b5273.png">
